src/ui/components/pages/exportAccounts/chooseAccounts.tsx

```diff
@@ -0,0 +1,133 @@
+                    .filter(([_, accounts]) => accounts.length !== 0)
+                    .map(([network, accounts]) => (
+                        <div key={network} className={styles.accountsGroup}>
+                            <header className={styles.accountsGroupHeader}>
+                                <i className={cn(styles.accountsGroupIcon, 'networkIcon')} />
+
+                                <h2 className={styles.accountsGroupLabel}>{networkLabels[network]}</h2>
+
+                                <input
+                                    checked={accounts.every((acc) => selected.has(acc.address))}
+                                    className={styles.checkbox}
+                                    type="checkbox"
+                                    onChange={(event) => {
+                                        const newChecked = event.currentTarget.checked;
+
+                                        setSelected((prevSelected) => {
+                                            const newSelected = new Set(prevSelected);
+
+                                            accounts.forEach((acc) => {
+                                                if (newChecked) {
+                                                    newSelected.add(acc.address);
+                                                } else {
+                                                    newSelected.delete(acc.address);
+                                                }
+                                            });
+
+                                            return newSelected;
+                                        });
+                                    }}
+                                />
+                            </header>
+
+                            <ul className={styles.accountList}>
+                                {accounts.map(({ address, name }) => (
+                                    <li key={address} className={styles.accountListItem} title={address}>
+                                        <div className={styles.accountInfo}>
+                                            <Avatar size={40} address={address} />
+
+                                            <div className={styles.accountInfoText}>
+                                                <div className={styles.accountName}>{name}</div>
+                                            </div>
+                                        </div>
+
+                                        <input
+                                            checked={selected.has(address)}
+                                            className={styles.checkbox}
+                                            name="selected"
+                                            type="checkbox"
+                                            value={address}
+                                            onChange={(event) => {
```

Comment:
This function seems to be very similar to the previous one. Maybe we can duplicate this?